engine-rest/engine-rest-openapi/src/main/templates/paths/history/job-log/get.ftl

```diff
@@ -0,0 +1,84 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/job-log/get-job-log-query/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "getHistoricJobLogs"
+      tag = "Historic Job Log"
+      summary = "Get Job Logs"
+      desc = "Queries for historic job logs that fulfill the given parameters.
+              The size of the result set can be retrieved by using the
+              [Get Job Log Count](${docsUrl}/reference/rest/history/job-log/get-job-log-query-count/)
+              method."
+  />
+
+  "parameters" : [
+
+    <#assign last = false >
+    <#assign requestMethod="GET"/>
```

Comment:
I cannot see that we use the request method in the `history-job-log-params.ftl`. Should we remove it here? It also does no harm as far as I can tell but I guess it's also not necessary?